tests/yast2_gui/yast2_control_center.pm

```diff
@@ -34,6 +34,7 @@ sub search {
     }
     else {
         send_key 'alt-s';
+        send_key 'backspace';
```

Comment:
why?

Reply:
This part doesn't work as expected, as when this method called with empty string, we expect that field is emptied then. But with old code we just navigate to the search field.